pages/en/lb3/Model-definition-JSON-file.md

```diff
@@ -431,6 +441,18 @@ For example:
         Property type. Can be any type described in <a href="LoopBack-types.html">LoopBack types</a>.
       </td>
     </tr>
+    <tr>
+      <td>
+        useDefaultIdType
+      </td>
+      <td>No</td>
+      <td>
+        Boolean
+      </td>
+      <td>
+        <strong>For auto-migrate usage</strong>. Set it to <code>false</code> when it's needed to auto-generated non-default type property values. e.g To use <code>uuid</code> as the id property, the id <code>type</code> should set to string, <code>generated</code> should set to true, and this field should set to false. Please check connectors' the <code>README</code> file for more information about auto-migration / auto-update.
```

Comment:
`needed to auto-generated` -> `needed to auto-generate`
`e.g To use` -> `For example, to use`
`should set to string` -> `should be set to string`
`should set to false` -> `should be set to false`
`Please check connectors' the <code>README</code> file` -> `Please check each connector's <code>README</code> file`
`auto-migration / auto-update.` -> `auto-migrate/auto-update.`